internal/plugins/golang/api.go

```diff
@@ -52,5 +63,21 @@ func (p *createAPIPlugin) Run() error {
 
 // SDK plugin-specific scaffolds.
 func (p *createAPIPlugin) run() error {
+	// Write CR paths to the samples' kustomization file. This file has a
+	// "do not modify" comment so it is safe to overwrite.
+	samplesKustomization := sampleKustomizationFragment
+	for _, gvk := range p.config.Resources {
+		samplesKustomization += fmt.Sprintf("- %s\n", makeCRFile(gvk))
+	}
+	kpath := filepath.Join("config", "samples")
+	if err := kustomize.Write(kpath, samplesKustomization); err != nil {
+		return err
+	}
 	return nil
 }
+
+// makeCRFile returns a Custom Resource example file name in the same format
+// as kubebuilder's CreateAPI plugin for a gvk.
+func makeCRFile(gvk config.GVK) string {
```

Comment:
Nit: Perhaps a more accurate name for this func. Or `crFileName()`
```suggestion
func getCRFileName(gvk config.GVK) string {
```